docs/UserGuide.md

```diff
@@ -50,55 +56,63 @@ WorkBook (WB) is a **desktop app for CS/tech students who are applying for inter
+| **Delete** | `delete INDEX`<br> e.g., `delete 3`                                                                                                                                                                   |
+| **List**   | `list`                                                                                                                                                                                                |
+| **Undo**   | `undo`                                                                                                                                                                                                |
+| **Redo**   | `redo`                                                                                                                                                                                                |
+| **Clear**  | `clear`                                                                                                                                                                                               |
+| **Help**   | `help`                                                                                                                                                                                                |
+| **Exit**   | `exit`                                                                                                                                                                                                |
 
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Note:**<br>
 
+* Commands without examples are considered trivial and can be executed by inputting the command without additional parameters.
+* Commands are **case-sensitive**!
+
+</div>
 
-### Adding an internship application: `add`
+### 4.1. Adding an internship application: `add`
 
 Adds an internship application to WorkBook.
 
 Format: `add c/COMPANY r/ROLE s/STAGE [d/DATE] [e/COMPANY_EMAIL] [l/LANGUAGE TAG] [t/TAG]…​`
 
 * Adds an internship application to the list in sorted order.
-* Fields not square-bracketed are mandatory.
-* Date must be properly formatted (DD-MMM-YYYY HH:mm) when inputted.
-* Date can represent the date it happened or the deadline of the corresponding `stage`
-* Email domain, if included, must be properly formatted (example@domain.com).
-* An application can have any number of tags and language tags and are added one by one (i.e.
-if you want 2 tags, you have to specify `t/two t/tags`).
+  * Note: Internship applications **in the past** are placed at the bottom.
+* Date can represent:
+    * The date it happened
+    * The deadline of the corresponding `Stage`
+* Date, if included, must be properly formatted `DD-MMM-YYYY HH:mm` when inputted.
```

Comment:
Also i think these notes about Date formatting should be shifted into the "things to note" section as it applies to the app as a whole and not the Add command.
Maybe the part about "Date can represent:" can be left here?

Reply:
https://ay2223s1-cs2103t-w13-3.github.io/tp/UserGuide.html you could get some inspiration from their "things to note" section, so maybe a header like theirs for the DateTime stuff and another Header for the Sorting stuff

Reply:
Nice find, might reference a table or 2 from there >:)